go.mod

```diff
@@ -3,6 +3,17 @@ module github.com/ElrondNetwork/elrond-deploy-go
 go 1.12
 
 require (
-	github.com/ElrondNetwork/elrond-go v1.0.77
```

Comment:
After `EN-5789-from-kyber-to-go-ed25519` is merged, `go.mod` should be changed again to point to a release tag